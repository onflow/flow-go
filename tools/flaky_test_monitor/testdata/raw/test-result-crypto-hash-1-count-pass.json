{"Time":"2021-09-24T11:27:29.121674-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_256"}
{"Time":"2021-09-24T11:27:29.121944-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_256","Output":"=== RUN   TestSanitySha3_256\n"}
{"Time":"2021-09-24T11:27:29.121969-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_256","Output":"--- PASS: TestSanitySha3_256 (0.00s)\n"}
{"Time":"2021-09-24T11:27:29.121973-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_256","Elapsed":0}
{"Time":"2021-09-24T11:27:29.121993-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_384"}
{"Time":"2021-09-24T11:27:29.121996-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_384","Output":"=== RUN   TestSanitySha3_384\n"}
{"Time":"2021-09-24T11:27:29.122-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_384","Output":"--- PASS: TestSanitySha3_384 (0.00s)\n"}
{"Time":"2021-09-24T11:27:29.122003-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha3_384","Elapsed":0}
{"Time":"2021-09-24T11:27:29.122006-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_256"}
{"Time":"2021-09-24T11:27:29.122126-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_256","Output":"=== RUN   TestSanitySha2_256\n"}
{"Time":"2021-09-24T11:27:29.122143-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_256","Output":"--- PASS: TestSanitySha2_256 (0.00s)\n"}
{"Time":"2021-09-24T11:27:29.122155-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_256","Elapsed":0}
{"Time":"2021-09-24T11:27:29.122161-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_384"}
{"Time":"2021-09-24T11:27:29.122168-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_384","Output":"=== RUN   TestSanitySha2_384\n"}
{"Time":"2021-09-24T11:27:29.122175-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_384","Output":"--- PASS: TestSanitySha2_384 (0.00s)\n"}
{"Time":"2021-09-24T11:27:29.122184-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanitySha2_384","Elapsed":0}
{"Time":"2021-09-24T11:27:29.122189-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanityKmac128"}
{"Time":"2021-09-24T11:27:29.122194-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanityKmac128","Output":"=== RUN   TestSanityKmac128\n"}
{"Time":"2021-09-24T11:27:29.122205-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanityKmac128","Output":"--- PASS: TestSanityKmac128 (0.00s)\n"}
{"Time":"2021-09-24T11:27:29.122212-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSanityKmac128","Elapsed":0}
{"Time":"2021-09-24T11:27:29.122216-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestHashersAPI"}
{"Time":"2021-09-24T11:27:29.122273-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestHashersAPI","Output":"=== RUN   TestHashersAPI\n"}
{"Time":"2021-09-24T11:27:29.122293-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestHashersAPI","Output":"    hash_test.go:114: math rand seed is 1632497249121800000\n"}
{"Time":"2021-09-24T11:27:29.122332-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestHashersAPI","Output":"--- PASS: TestHashersAPI (0.00s)\n"}
{"Time":"2021-09-24T11:27:29.122339-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestHashersAPI","Elapsed":0}
{"Time":"2021-09-24T11:27:29.122344-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3"}
{"Time":"2021-09-24T11:27:29.122351-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3","Output":"=== RUN   TestSha3\n"}
{"Time":"2021-09-24T11:27:29.122663-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3","Output":"    hash_test.go:158: math rand seed is 1632497249122032000\n"}
{"Time":"2021-09-24T11:27:29.122678-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_256"}
{"Time":"2021-09-24T11:27:29.122724-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_256","Output":"=== RUN   TestSha3/SHA3_256\n"}
{"Time":"2021-09-24T11:27:29.223618-04:00","Action":"run","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_384"}
{"Time":"2021-09-24T11:27:29.223646-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_384","Output":"=== RUN   TestSha3/SHA3_384\n"}
{"Time":"2021-09-24T11:27:29.347325-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3","Output":"--- PASS: TestSha3 (0.23s)\n"}
{"Time":"2021-09-24T11:27:29.347372-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_256","Output":"    --- PASS: TestSha3/SHA3_256 (0.10s)\n"}
{"Time":"2021-09-24T11:27:29.347376-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_256","Elapsed":0.1}
{"Time":"2021-09-24T11:27:29.347401-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_384","Output":"    --- PASS: TestSha3/SHA3_384 (0.12s)\n"}
{"Time":"2021-09-24T11:27:29.347405-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3/SHA3_384","Elapsed":0.12}
{"Time":"2021-09-24T11:27:29.347408-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Test":"TestSha3","Elapsed":0.23}
{"Time":"2021-09-24T11:27:29.347412-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Output":"PASS\n"}
{"Time":"2021-09-24T11:27:29.348371-04:00","Action":"output","Package":"github.com/onflow/flow-go/crypto/hash","Output":"ok  \tgithub.com/onflow/flow-go/crypto/hash\t0.349s\n"}
{"Time":"2021-09-24T11:27:29.348417-04:00","Action":"pass","Package":"github.com/onflow/flow-go/crypto/hash","Elapsed":0.349}