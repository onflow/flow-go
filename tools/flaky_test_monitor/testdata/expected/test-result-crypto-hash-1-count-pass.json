{
  "commit_sha": "46baf6c6be29af9c040bc14195e195848598bbae",
  "commit_date": "2021-09-21T18:06:25+00:00",
  "job_run_date": "2021-09-21T21:06:25+00:00",
  "results": [
    {
      "package": "github.com/onflow/flow-go/crypto/hash",
      "result":"pass",
      "elapsed": 0.349,
      "output": [
        "PASS\n",
        "ok  \tgithub.com/onflow/flow-go/crypto/hash\t0.349s\n"
      ],
      "tests": [
        {
          "test": "TestSanitySha3_256",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSanitySha3_256\n",
            "--- PASS: TestSanitySha3_256 (0.00s)\n"
          ],
          "result": "pass",
          "elapsed": 0
        },
        {
          "test": "TestSanitySha2_256",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSanitySha2_256\n",
            "--- PASS: TestSanitySha2_256 (0.00s)\n"
          ],
          "result": "pass",
          "elapsed": 0
        },
        {
          "test": "TestSanitySha3_384",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSanitySha3_384\n",
            "--- PASS: TestSanitySha3_384 (0.00s)\n"
          ],
          "result": "pass",
          "elapsed": 0
        },
        {
          "test":"TestSanitySha2_384",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSanitySha2_384\n",
            "--- PASS: TestSanitySha2_384 (0.00s)\n"
          ],
          "result": "pass",
          "elapsed": 0
        },
        {
          "test":"TestSanityKmac128",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSanityKmac128\n",
            "--- PASS: TestSanityKmac128 (0.00s)\n"
          ],
          "result": "pass",
          "elapsed": 0
        },
        {
          "test":"TestHashersAPI",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestHashersAPI\n",
            "    hash_test.go:114: math rand seed is 1632497249121800000\n",
            "--- PASS: TestHashersAPI (0.00s)\n"
          ],
          "result": "pass",
          "elapsed": 0
        },
        {
          "test":"TestSha3",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSha3\n",
            "    hash_test.go:158: math rand seed is 1632497249122032000\n",
            "--- PASS: TestSha3 (0.23s)\n"
          ],
          "result": "pass",
          "elapsed": 0.23
        },
        {
          "test": "TestSha3/SHA3_256",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSha3/SHA3_256\n",
            "    --- PASS: TestSha3/SHA3_256 (0.10s)\n"
          ],
          "result": "pass",
          "elapsed": 0.1
        },
        {
          "test": "TestSha3/SHA3_384",
          "package": "github.com/onflow/flow-go/crypto/hash",
          "output": [
            "=== RUN   TestSha3/SHA3_384\n",
            "    --- PASS: TestSha3/SHA3_384 (0.12s)\n"
          ],
          "result": "pass",
          "elapsed": 0.12
        }
      ]
    }
  ]
}